examples/output/2island/gfwValueOverlap.json

```diff
@@ -232,15 +232,35 @@
       "sketchId": "17853",
       "classId": "gear_type:set-longlines",
       "groupId": null,
-      "geographyId": null,
+      "geographyId": "nearshore",
       "value": 19.248055,
       "extra": {
         "sketchName": "santamaria"
       }
     }
   ],
   "sketch": {
-    "id": 17851,
+    "properties": {
+      "2991": "2island",
+      "id": "17851",
+      "name": "2island",
+      "postId": null,
+      "userId": "531",
+      "userSlug": "Abby",
+      "createdAt": "2023-08-03T23:23:31.444221+00:00",
+      "updatedAt": "2023-08-03T23:24:36.079995+00:00",
+      "collectionId": null,
+      "isCollection": true,
+      "sharedInForum": false,
+      "sketchClassId": "213",
+      "userAttributes": null
+    },
+    "bbox": [
+      -25.370367244,
+      36.93442097,
+      -24.90137653,
+      37.932012737
+    ],
```

Comment:
These weren't changes, the order of things in the file just changed.  This may be due to running on my Windows Ubuntu system, not sure.